libs/base/ui/form/multi-range/src/multi-range.styles.ts

```diff
@@ -31,7 +32,12 @@ export const multiRangeStyles = css`
     margin: 9px 0;
   }
 
-  :host > * {
+  :host([invalid]) {
+    display: none;
```

Comment:
So we are just hiding the whole component in case something is invalid? I'd say it's against common behaviour for html components.

Probably we will need to align on our own strategy for such cases. I'll discuss it on AA catchup.

Reply:
refactored